diagnostic.js

```diff
@@ -32,7 +45,33 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update(
+  [
+    { house:"House Stark" },
+    { members:
+      [
+        name: "Ned Stark",
+        name: "Arya Stark",
+        name: "Sansa Stark",
+      ]
+    },
+    { house:"House Targaryen"},
+    {
+      members:
+      [
+        name: "Viserys Targaryen",
+        name: "Daenerys Targaryen"
+      ]
+    },
+    { house:"House Arryn"},
+    {
+      members:
+      [
+        name: "Jon Arryn"
+      ]
+    }
+  ]
+)
```

Comment:
This is not correct. With `update`, the 1st argument is the thing we are searching by. If you are not using a method like $set or $push, that object will get replaced by the thing in the 2nd argument.